src/blueprint/into_schema.rs

```diff
@@ -48,13 +48,20 @@ fn to_type(def: &Definition) -> dynamic::Type {
             }
             Some(expr) => {
               let expr = expr.to_owned();
+              let field_of_type = type_ref.clone();
```

Comment:
Why do we need these changes?

Reply:
The problem was that for the '/users&id=?` request the endpoint was returning an array but the GQL schema was expecting a single object, therefore returning an error to the GQL query.
 Before making this change, the request would go through the batched DataLoader that would take the array and process it, but now the response would only be processed by this future field resolver that did not expect to receive an array. 
 
My approach to this was to check the expected field type, and if was a single object but the response from the REST API was an array, extract the object from the array. Hope this is the right place to do it, if not please let me know how to approach this, thanks! 

This specific line was added because i needed the type of the field in a `match` context.

Reply:
Ideally this should not be done. It should fail as wrong URL is configured.